Source/Applications/SystemCenter/Model/Configuration/APIConfiguration.cs

```diff
@@ -44,22 +37,38 @@ public class APIConfiguration
         [Setting]
         [DefaultValue("http://localhost:8989")]
         [SettingName("Url")]
-        public string Host { get; set; }
+        public string OpenXDAHost { get; set; }
```

Comment:
This seems uneccesarry. Either your Setting is `XDAAPISettings.Host` or `setting.OpenXDAHost` but `XDAAPISettings.OpenXDAHost` is redundant.
I would revert this back to `Host` and `Key` since the `SettingName` attribute already distinguishes `MiMD.Host` and `OpenXDA.Host` in the `Setting` Table